generate/generate.go

```diff
@@ -39,180 +39,17 @@ func New() Generator {
 			OS:   runtime.GOOS,
 			Arch: runtime.GOARCH,
 		},
-		Root: rspec.Root{
-			Path:     "",
```

Comment:
Because option --rootfs-path has default value `rootfs`, then I think keep it or not doesn't matter , so removed it.